Airwallex/Core/Sources/Internal/Extensions/UIViewController+utils.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UIViewController+utils.swift
+//  Core
+//
+//  Created by Tony He (CTR) on 2024/7/23.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+@objc
+public extension UIViewController {
+    static var activityIndicator: UIActivityIndicatorView?
```

Comment:
This is not recommend. The class variable can be shared among multiple VCs, which can be problematic when VCs show on the same page.